PIL/ImageDraw.py

```diff
@@ -91,41 +91,6 @@ def __init__(self, im, mode=None):
         self.font = None
 
     ##
-    # Set the default pen color.
-
-    def setink(self, ink):
-        # compatibility
-        if warnings:
-            warnings.warn(
-                "'setink' is deprecated; use keyword arguments instead",
-                DeprecationWarning, stacklevel=2
-                )
-        if isStringType(ink):
-            ink = ImageColor.getcolor(ink, self.mode)
-        if self.palette and not isinstance(ink, numbers.Number):
-            ink = self.palette.getcolor(ink)
-        self.ink = self.draw.draw_ink(ink, self.mode)
-
-    ##
-    # Set the default background color.
-
-    def setfill(self, onoff):
-        # compatibility
-        if warnings:
-            warnings.warn(
-                "'setfill' is deprecated; use keyword arguments instead",
-                DeprecationWarning, stacklevel=2
-                )
-        self.fill = onoff
-
-    ##
-    # Set the default font.
-
-    def setfont(self, font):
```

Comment:
It's been marked as deprecated in the docs since 1.1.5. I figured it was a sufficient length of time for even a docs warning to be effective. I'll restore the function and add a deprecation warning.